login.css

```diff
@@ -51,8 +50,8 @@ nav a{
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family: 'Roboto', sans-serif;
-	font-weight:300;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
20/08/2021: Changing the fonts style for nav to be Open Sans and set the link in the nav to bold